automatic/mpc-hc/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,27 @@
+$ErrorActionPreference = 'Stop'
```

Comment:
Why adding an uninstall script?
The uninstaller is created by inno setup which should be detected automatically by chocolateys Auto Uninstaller